tests/Unit/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Test_DampedHarmonicGaugeQuantities.cpp

```diff
@@ -0,0 +1,3176 @@
+
+  // Initialize settings
+  const double amp_coef_L1 = pdist(generator);
+  const int exp_L1 = idist(generator);
+  // roll on function parameters for lapse / shift terms
+  const double t_start_L1 = pdist(generator) * 0.1;
+  const double sigma_t_L1 = pdist(generator) * 0.2;
+  const double r_max = pdist(generator) * 0.7;
+  const double roll_on_L1 =
+      GeneralizedHarmonic::DampedHarmonicGauge_detail::roll_on_function(
+          t, t_start_L1, sigma_t_L1);
+  const auto& weight =
+      GeneralizedHarmonic::DampedHarmonicGauge_detail::weight_function<
+          SpatialDim, Frame::Inertial, DataVector>(x, r_max);
+
+  const auto gauge_h_init = make_with_value<typename db::item_type<
+      GeneralizedHarmonic::Tags::InitialGaugeH<SpatialDim, Frame::Inertial>>>(
+      x, 0.);
+  typename db::item_type<
+      GeneralizedHarmonic::Tags::GaugeH<SpatialDim, Frame::Inertial>>
+      gauge_h{};
+  GeneralizedHarmonic::damped_harmonic_h<SpatialDim, Frame::Inertial>(
+      make_not_null(&gauge_h), gauge_h_init, lapse, shift,
+      sqrt_det_spatial_metric, spacetime_metric, t, x, amp_coef_L1, 0., 0.,
+      exp_L1, 0, 0, /* exponents */
+      /* roll on function parameters for lapse / shift terms */
+      -1., -1., t_start_L1, sigma_t_L1, -1., -1., -1., -1.,
+      r_max /* weight function */);
+
+  /*
+   * Evaluate analytic solution for Schwarzschild, ab initio.
+   */
+  const double mass = solution.mass();
+  const std::array<double, SpatialDim> center = solution.center();
+  // 1) shift coords with BH at the center
+  auto x_minus_center = x;
+  // DataVector r_squared(get_size(x), 0.);
+  auto r_squared = get<0>(x);
+  r_squared -= get<0>(x);
+  for (size_t i = 0; i < SpatialDim; ++i) {
+    x_minus_center.get(i) -= gsl::at(center, i);
+    r_squared += square(x_minus_center.get(i));
+  }
+  const DataVector r_ = sqrt(r_squared);
+  // 2) compute scalar function that defines KerrSchild coords
+  const DataVector H = mass * (1. / r_);
+  const DataVector H2 = H * 2.;
+  // 3) compute lapse
+  const Scalar<DataVector> lapse_ab_initio{sqrt(1. / (1. + H2))};
+  // 4) compute 3-metric
+  auto spatial_metric_ab_initio =
+      make_with_value<tnsr::ii<DataVector, SpatialDim, Frame::Inertial>>(x, 0.);
+  for (size_t i = 0; i < SpatialDim; ++i) {
+    for (size_t j = i; j < SpatialDim; ++j) {
+      spatial_metric_ab_initio.get(i, j) =
+          H2 * x_minus_center.get(i) * x_minus_center.get(j) / r_squared;
+      if (i == j) {
+        spatial_metric_ab_initio.get(i, i) += 1.;
+      }
+    }
+  }
+  const auto& spacetime_unit_normal_one_form_ab_initio =
+      gr::spacetime_normal_one_form<SpatialDim, Frame::Inertial, DataVector>(
+          lapse_ab_initio);
+  const auto& det_spatial_metric_ab_initio =
+      determinant_and_inverse(spatial_metric_ab_initio).first;
+  Scalar<DataVector> sqrt_det_spatial_metric_ab_initio(
+      sqrt(get(det_spatial_metric_ab_initio)));
+  /* *** */
```

Comment:
what is this comment all about? Could you please remove it?

Reply:
removed